Objects/unicodeobject.c

```diff
@@ -14309,9 +14309,9 @@ unicode_getnewargs(PyObject *v, PyObject *Py_UNUSED(ignored))
 }
 
 /*
-This function searchs the longest common leading whitespace
+This function searches the longest common leading whitespace
 of all lines in the [src, end).
-It returns the length of the common leading whitespace and sets `output` to
+It returns the length of the common leading whitespace and sets *output* to
```

Comment:
Revert this. We don't use md-like syntax in C as it can be confused with pointers.